style.css

```diff
@@ -17,13 +17,71 @@ header nav {
   align-items: center;
   justify-content: space-between;
   width: 100%;
+  gap: 25%;
+}
+
+/* Menu changes */
+.menu-icon {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  left: 0;
+  right: 0;
+  top: 0;
+  background-color: #6070fff0;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(10px);
+  text-align: left;
+  display: none;
+}
+
+.item-closed {
+  margin-left: 280px;
+}
+
+.menu-icon .close {
+  padding-top: 50px;
+  text-align: right;
+  padding-right: 38px;
+  font-weight: lighter;
+}
+
+.mobile-nav {
+  padding: 15px 28px;
 }
 
 header a {
   margin: 0 12px;
   font-size: 18px;
 }
 
+nav a {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 20px;
+  text-decoration: none;
+  width: 250px;
+  height: 20px;
+  color: #6070ff;
+  display: flex;
+}
```

Comment:
- [ ] Your mobile menu looks good, but the width of this element is making it possible for the user to scroll horizontally, leaving whitespace on the side and making your page not look professional. Kindly remove this extra width to make the app look like the Figma template.
 - Your design:
 - 
<img width="462" alt="image" src="https://user-images.githubusercontent.com/61919617/215531386-0137e3f6-3cfa-4760-90bf-2a91e1558481.png">